crates/flux-desugar/src/desugar.rs

```diff
@@ -54,6 +54,16 @@ pub(crate) fn desugar_qualifier<'genv>(
     })
 }
 
+fn path_expr<'genv>(
```

Comment:
I don't think any of the changes in this file are needed. Please revert them

Reply:
In this case the `.span` is indeed just that of the identifier -- have renamed the field to `ident_span` to clarify.